ddapm_test_agent/apmtelemetry.py

```diff
@@ -0,0 +1,13 @@
+import json
+from typing import Any
+from typing import Dict
+from typing import cast
+
+
+TelemetryEvent = Dict[str, Any]
```

Comment:
would be really cool to flesh out this schema, but all good for now!

Reply:
yeah, I haven't decided yet, but since we have an OpenAPI schema, we can play with how we introduce that. is it a check or part of this decoding?